typings/message.d.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference types="node" />
+
+import { CreateTypeDesc } from "./types";
+
+export declare class Section {
+  typecode: number;
+  content: any;
+  multiple: any;
+  constructor(typecode: number, content: any, multiple: any);
+  described(item: any): any;
+}
+
+export declare interface Message {
+  [x: string]: any;
+  toJSON: () => {
+    [x: string]: any;
+  };
+  inspect: () => string;
+  toString: () => string;
+}
+
+export interface message {
+  data_section: (data: any) => Section;
+  sequence_section: (list: any) => Section;
+  data_sections: (data_elements: any) => Section;
+  sequence_sections: (lists: any) => Section;
+  encode: (msg: any) => any;
+  decode: (buffer: Buffer) => Message;
```

Comment:
While implementing the batch send, I found the encode function useful. It would be nice to correctly define the types of these functions. It would be helpful to users.